LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I thought we wanted apache?

Reply:
Ah, it was a mistake when I wrote "Apache-2.0" in the PR description; the license included is GPLv3 since that is the Emacs convention.

Since the copyright is in your name, the software can be relicensed at your discretion. If we want to use Apache-2.0, I'll need to check with Melpa (the largest Emacs package archive) to see if this is acceptable. In order for the package to be included in Melpa, it must be "GPL compatible", and Apache-2.0 is compatible with GPLv3 but incompatible with GPLv2, so it sorta depends on what they mean by "GPL compatible" 🤷

Reply:
Sure, we can do that :grin: